src/components/OmouComponents/IdleLogout.js

```diff
@@ -131,44 +146,37 @@ const IdleLogout = () => {
             }, 900000);
         });
     };
-
-    useEffect(() => {
-        (async () => {
-            await refreshTokenAfter15Minutes();
-        })();
-
-    }, []);
     
+
     const logoutAndCloseModal = () => {
         handleClose();
         handleLogout();
     };
 
+    // Opens the modal, resets the modal timeout ref
     async function handleOpen() {
         setOpenModal(true);
-        handleReset();
-        setRemainingMsUntilPrompt(modalTimeout);
-
-        sessionTimeoutRef.current = setTimeout(logoutAndCloseModal, 5 * 1000);
+        sessionTimeoutRef.current = setTimeout(logoutAndCloseModal, modalTimeout);
     }
     
+    // Close the modal, clear the modal timeout ref, reset the token
```

Comment:
use comments to explain why you're doing something - it's clear from the function that you're clearInterval(), setOpenModal(false) and resetToken(), but it's not clear why and when this function is used.